### PR TITLE
Ftr/117 remove leaflet

### PR DESCRIPTION
**Pull request by daniel-jettka**
## Description, Context and related Issue

Removed all leaflet files and content because Edirom does not support leaflet anymore.

Refs #117 

## How Has This Been Tested?

(...)

## Types of changes

- cleanup

### CODE REVIEW

**Comment by daniel-jettka** [comment]
@bwbohl Could you have a look at this? I am not sure how to check if everything's working fine after removing all leaflet content - I mean it wasn't in use anyway, right?

**Review by peterstadler — COMMENTED** [review]
I was trying to test this PR with the clarinet quintet data I grabbed from the [pipeline build](https://git.uni-paderborn.de/wega/klarinettenquintett-edirom/-/jobs/328551).

First I was hit by the error #443 reported by @roewenstrunk so I had to merge his fix into  this branch. Then I was able to take a look and it all seemed to work (i.e. I did no systematic tests).

**Comment by peterstadler** [comment]
I rebased this branch to bring it up to date with current dev. Please make sure to
1. do a `git fetch` to fetch the changes, and
2. do a `git reset --hard origin/ftr/117-remove-leaflet` to point your HEAD at the force-pushed changes

File: add/index.xql [hunk 1]
```diff
@@ -61,13 +61,6 @@ let $eoIndexPage :=  <html>
                             <!-- **Raphael JS** -->
                             <script type="text/javascript" src="resources/js/raphael-min.js"/>
                             
-                            <!-- **Leaflet** -->
-                            <link rel="stylesheet" href="resources/leaflet-0.7.3/leaflet.css"/>
-                            <link rel="stylesheet" href="resources/Leaflet.tooltip-master/dist/leaflet.tooltip.css"/>
-                            <script type="text/javascript" src="resources/leaflet-0.7.3/leaflet.js"/>
-                            <script type="text/javascript" src="resources/facsimileLayer/FacsimileLayer.js"/>
```
Comment (peterstadler) [inline add/index.xql:68]: is `FacsimileLayer.js` not needed anymore?

File: app/view/window/text/FacsimileView.js [hunk 2]
```diff
@@ -23,7 +23,6 @@ Ext.define("EdiromOnline.view.window.text.FacsimileView", {
     requires: [
         "EdiromOnline.view.window.image.ImageViewer",
         "EdiromOnline.view.window.image.OpenSeaDragonViewer",
```
Comment (peterstadler) [inline app/view/window/text/FacsimileView.js:25]: Comma should be removed:
```suggestion
        "EdiromOnline.view.window.image.OpenSeaDragonViewer"
```

File: add/index.xql (hunk 1 above)
Comment (daniel-jettka) [reply add/index.xql:68]: Good question, a `grep -R FacsimileLayer` returns only occurences in resources/facsimileLayer/FacsimileLayer.js

Is that sufficient to evaluate if it is not needed anymore? @roewenstrunk or anyone...

(N.B. resources/facsimileLayer/FacsimileLayer.js should also be removed if not needed anymore)

**Comment by peterstadler** [comment]
I see that `FacsimileLayer.js` got removed now but I still found some references in 
https://github.com/Edirom/Edirom-Online/blob/7b21287f4cdda72c6f0a393c503fa72e45c84a7d/index.html#L38-L43

File: add/index.xql (hunk 1 above)
Comment (peterstadler) [reply add/index.xql:68]: pinging @roewenstrunk to have a look and formally assigning him as reviewer

Comment (roewenstrunk) [reply add/index.xql:68]: This is only needed in the Leaflet world. It extends Leaflet technology

**Review by roewenstrunk — APPROVED** [review]
Everything works with my data. The changes all seem correct.

**Pull request merged**